go.mod

```diff
@@ -172,3 +171,10 @@ require (
 	sigs.k8s.io/kustomize/kyaml v0.13.0 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
 )
+
+replace (
+	// FIXME: remove this pin when knative/eventing#6451 is merged
+	knative.dev/eventing => github.com/cardil/knative-eventing v0.11.1-0.20220718191420-28d8469b82bf
+	// FIXME: remove this pin when knative-sandbox/reconciler-test#294 is merged
+	knative.dev/reconciler-test => github.com/cardil/knative-reconciler-test v0.0.0-20220715181934-169264f8ba23
```

Comment:
Remove this pin, after merge of https://github.com/knative-sandbox/reconciler-test/pull/294

/hold

Reply:
Fixed in b85f141be77dee08942c478fc6c19f85d885d5a3

/unhold